src/app/organization/working-days/working-days.component.html

```diff
@@ -0,0 +1,48 @@
+<div class="container">
+
+  <mat-card>
+
+    <form [formGroup]="workingDaysForm" (ngSubmit)="submit()">
+
+      <mat-card-content>
+
+        <div fxLayout="column">
+
+          <div fxLayout="row" fxLayout.lt-md="column">
+            <mat-label fxFlex="50%" class="mat-h4">Working Days</mat-label>
+            <div fxFlex="50%" fxLayout="column" formArrayName="recurrence">
+              <div *ngFor="let day of recurrence.controls; let i = index;">
+                <mat-checkbox labelPosition="after" [formControlName]="i">
+                  {{ weekDays[i].name }}
+                </mat-checkbox>
+              </div>
+            </div>
+          </div>
+
+          <mat-form-field>
+            <mat-label>Payments due on non working days</mat-label>
+            <mat-select formControlName="repaymentRescheduleType">
+              <mat-option *ngFor="let repaymentRescheduleType of repaymentRescheduleTypeData" [value]="repaymentRescheduleType.id">
+                {{ repaymentRescheduleType.value }}
+              </mat-option>
+            </mat-select>
+          </mat-form-field>
+
+            <mat-checkbox labelPosition="before" formControlName="extendTermForDailyRepayments">
```

Comment:
```suggestion
          <mat-checkbox labelPosition="before" formControlName="extendTermForDailyRepayments">
```
Indent properly, remove extra space.